### DISCUSSION TITLE
Upgrading 7 in chain..

### DISCUSSION THREAD

**Discussion by Pan05** [soundshed/soundshed-app#39] [Q&A]
Hi there,

With the recent announcement of an EQ pedal, which is great news, I am also a bit concerned that this replaces one of the 7 pedal/ amp positions. I am wondering if it's possible to ever have over 7 parameters in the chain.. or to mix and match pedals, for instance 2 delays instead of being limited to 1. I know that a looper is out of the question as there is no memory to make this possible, but from a technical perspective would it be possible to overcome the chain of 7 predesignated slots? If so I really hope either PG or Soundshed could help. 

Thanks for your help and great work!

**Comment by webprofusion-chrisc** [comment]
Hi, presets don't specify a slot so as far as I know there is no may to manipulate which fx are associated with which map slot, instead the amp knows which fx fit in which slot.

The relevant code for sending the present is here, but I haven't tried manipulating any of that to try to get different results:
https://github.com/soundshed/soundshed-app/blob/main/src/spork/src/devices/spark/sparkCommandMessage.ts#L349
